Kitodo-DataManagement/src/main/java/org/kitodo/data/interfaces/WorkflowInterface.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.data.interfaces;
+
+import java.util.Objects;
+
+import org.apache.commons.lang3.StringUtils;
+import org.kitodo.data.database.enums.WorkflowStatus;
+
+public interface WorkflowInterface extends DataInterface {
+
+    /**
+     * Returns the label of the workflow.
+     *
+     * @return the label
+     */
+    String getTitle();
+
+    /**
+     * Sets the label of the workflow.
+     *
+     * @param title
+     *            label to set
+     */
+    void setTitle(String title);
+
+    /**
+     * Returns the stage of the workflow. Statuses:
+     *
+     * <dl>
+     * <dt>DRAFT</dt>
+     * <dd>the workflow is being created</dd>
+     * <dt>ACTIVE</dt>
+     * <dd>the workflow is in use</dd>
+     * <dt>ARCHIVED</dt>
+     * <dd>the workflow is no longer used</dd>
+     * </dl>
+     *
+     * @return the stage
+     * @deprecated Use {@link #getStatus()}.
+     */
+    @Deprecated
+    default String getWorkflowStatus() {
+        WorkflowStatus status = getStatus();
+        return Objects.nonNull(status) ? status.toString() : "";
+    }
+
+    /**
+     * Returns the stage of the workflow. Statuses:
+     *
+     * <dl>
+     * <dt>DRAFT</dt>
+     * <dd>the workflow is being created</dd>
+     * <dt>ACTIVE</dt>
+     * <dd>the workflow is in use</dd>
+     * <dt>ARCHIVED</dt>
+     * <dd>the workflow is no longer used</dd>
+     * </dl>
+     *
+     * @return the stage
+     */
+    WorkflowStatus getStatus();
+
+    /**
+     * Sets the stage of the workflow. One of {@link WorkflowStatus}.
+     *
+     * @param status
+     *            as String
+     * @throws IllegalArgumentException
+     *             if {@link WorkflowStatus} has no constant with the specified
+     *             name
+     * @deprecated Use {@link #setStatus(WorkflowStatus)}.
+     */
+    @Deprecated
+    default void setWorkflowStatus(String status) {
+        setStatus(StringUtils.isNotBlank(status) ? WorkflowStatus.valueOf(status) : null);
```

Comment:
`StringUtils.isBlank` and `StringUtils.isNotBlank` are checking for white spaces. Is this white space check needed? If not `StringUtils.isEmpty` and ` StringUtils.isNotEmpty` is much faster.

The maybe wrong usage is in more places added but I'm writing this only once.

Maybe through my previous comment for the log4j String usage and a wrong example the usage is here used. Sorry, my fault :-(